Automattic/wp-calypso
# Store Orders: Create a new component for the "Total Rows" on the single order view

This PR creates a new component, `OrderTotalRow`, which handles displaying all the "totals" on the single order view. That's these lines, "Discount", "Shipping", etc:

<img width="270" alt="screen shot 2017-10-06 at 4 21 59 pm" src="https://user-images.githubusercontent.com/541093/31297047-8932f206-aab2-11e7-97eb-c692eed8d91e.png">

This change also touches this part, in the refunds dialog:

<img width="268" alt="screen shot 2017-10-06 at 4 30 55 pm" src="https://user-images.githubusercontent.com/541093/31297395-d6d7e66e-aab3-11e7-8311-e898c293ba1b.png">

There should be no visual changes here. This PR chains off #18632, and also continues a refactor of the refund dialog + table, so that the order can be editable as a separate act from giving a refund.

**To test**

- View an order, make sure the totals (discount, shipping, total, refunded) all display correctly (Note: refund should only show if there is a refund)
- Click to open the refund modal, make sure the data is correctly displayed, and changing the shipping amount works
- Submit the refund, make sure that succeeds.
